### DISCUSSION TITLE
Image source not readable

### DISCUSSION THREAD

**Discussion by Yiddishe-Kop** [spatie/laravel-medialibrary#2479] [Q&A]
Laravel 8
PHP 8
spatie/laravel-medialibrary": "^9.5.5",

When using a watermark I get an error : `Image source not readable`.

I noticed that if I remove the `watermarkOpacity` it works fine.

My sample code:

```php
$this->addMediaConversion('thumb')
            ->width(500)
            ->watermark(storage_path('app/watermark.png'))
            ->watermarkOpacity(50)
            ->watermarkPadding(15, 15)
            ->watermarkPosition(Manipulations::POSITION_CENTER)
            ->watermarkHeight(80, Manipulations::UNIT_PERCENT)
            ->watermarkWidth(80, Manipulations::UNIT_PERCENT)
            ->performOnCollections('image');
```

---

I'm aware of [this discussion](https://github.com/spatie/image/discussions/129) & Intervention/image#1080, but as I noticed it only happens when using the `watermark`, I thought I'd open the issue here. Hope I'm not bothering.

Thanks!

**Comment by guysolamour** [comment]
I have the same issue too

**Comment by samerzaki** [comment]
I have the same issue too

**Comment by echocage123** [comment]
same issue
